README.md

```diff
@@ -4,30 +4,22 @@ Pipeline that processes raw Direct Infusion Mass Spectrometry data.
 ### Folder Structure
 ```
 .
-|───GUI/ (GUI scripts)
+|───CustomModules/ (GitHub repo with pipeline scripts)
+|───assests/ (extra Nextflow files)
 |───db/ (Human Metabolome Database files)
-|───extra/ (flowcharts)
-|───pipeline/ (pipeline scripts)
-|───post/ (scripts that can be manually run after pipeline)
 ```
 
-## Setup GUI
-Used R version: 3.6.1 \
-Libraries: DT, shiny, shinydashboard, shinyFiles, ssh
-
-- Copy config_default.R to your own config.R, and configure as needed.
-
 ## Docker image 
 ```
-docker build -t umcugenbioinf/dims:[tag] -f Dockerfile . 
-docker push umcugenbioinf/dims:[tag]
+docker build -t ghcr.io/umcugenetics/[NAME]:[tag] .
+docker push ghcr.io/umcugenetics/[NAME]:[tag]
 ```
 
 on HPC: 
 ```
 srun -c 2 -t 0:30:00 -A dbg_mz --mem=100G --gres=tmpspace:100G --pty /usr/bin/bash 
 cd /hpc/dbg_mz/tools/singularity_cache/ 
-singularity build /hpc/dbg_mz/tools/singularity_cache/dims-[tag].img docker://umcugenbioinf/dims:[tag]
+singularity build /hpc/dbg_mz/tools/MAP/NAME.img docker://ghcr.io/umcugenetics/[NAME]:[tag]
```

Comment:
/hpc/dbg_mz/tools/MAP/ does not exist on HPC. Is MAP a variable that needs to be changed by the user? Maybe include [MAP] /< MAP> and explain a bit about these variables

Reply:
Thanks, this was a copy error and has been changed.